src/iiiftilesource.js

```diff
@@ -444,11 +444,10 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     /**
      * Determine whether arbitrary tile requests can be made against a service with the given profile
      * @function
-     * @param {array} options - IIIF profile array
-     * @param {Array|String} options.profile
+     * @param {Object} options - IIIF option
+     * @param {String} options.profile
```

Comment:
Actually, for `options.profile` the `Array|String` was correct because it can be either an array or a string.